README.md

```diff
@@ -59,3 +104,24 @@ As a host,
 So that I don’t lose out on bookings
 I would like my property to remain available until confirmed
 ```
+-----
+
+## Technologies
+- Ruby
+- Sinatra
+- Capybara
+- Rake | Active Record
+- PostgreSQL
+- HTML | CSS
+- SimpleCov
+- Rubocop
```

Comment:
I would add BCrypt as well for authentication, and maybe Git itself for version control